### PR TITLE
Openstack lbclasses

### LINKED ISSUE #446: Define FIP subnet pool in shoot creation dialog
OpenStack cloud profile can contain the following loadbalancer classes:

```yaml
      floatingPools:                                                                                                                                                                                                                               
      - loadBalancerClasses:                                                                                                                                                                                                                       
        - floatingSubnetID: cc4cf15d-dcd3-43bb-bc31-bfd4b9d6996d                                                                                                                                                                                   
          name: internal
        - floatingSubnetID: 1bef59b2-74a5-4f94-aad6-83a59abe2aa0
          name: internet
        name: FloatingIP-external-cp
      - loadBalancerClasses:
        - floatingSubnetID: 42f12acd-244d-486e-8bbf-c58b0cb44619
          name: default
        name: FloatingIP-external-cp-gardener
```

Shoot spec must allow to define the desired subnets, e.g.:

```yaml
spec:
  addons:
    kubernetes-dashboard:
      authenticationMode: basic
      enabled: true
    nginx-ingress:
      enabled: true
  cloud:
    openstack:
      floatingPoolName: FloatingIP-external-cp-gardener
      loadBalancerClasses:
      - floatingSubnetID: d2eda633-e930-464e-b285-ecb826e02861
        name: default
      loadBalancerProvider: f5networks
```

### PR DESCRIPTION
**Pull request by grolu**
**What this PR does / why we need it**:

Screen Shot 2019-12-03 at 13 18 25

Screen Shot 2019-12-03 at 13 18 36

**Which issue(s) this PR fixes**:
Fixes #446

**Special notes for your reviewer**:

**Release note**:

```improvement user
OpenStack: Show available load balancer classes on the cluster details page
```

### CODE REVIEW

File: frontend/src/components/ShootDetails/LbClass.vue [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+limitations under the License.
+-->
+
+<template>
+  <g-popper
+    :title="lbClass.name"
+    toolbarColor="cyan darken-2"
+    :popperKey="`lbclass_${lbClass.name}`"
+  >
+    <span slot="content-before">
```
Comment (holgerkoser) [inline frontend/src/components/ShootDetails/LbClass.vue]: depends on other PR

File: frontend/src/components/ShootDetails/LbClass.vue [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  props: {
+    lbClass: {
+      type: Object
+    }
+  }
+}
+</script>
+
+<style lang="styl" scoped>
+  .cursor-pointer >>> .v-chip__content {
```
Comment (holgerkoser) [inline frontend/src/components/ShootDetails/LbClass.vue]: use class if possible?

File: frontend/src/components/ShootDetails/ShootInfrastructureCard.vue [hunk 3]
```diff
@@ -98,6 +98,21 @@ limitations under the License.
...
 
+      <template v-if="!!shootLoadbalancerClasses">
+        <v-divider class="my-2" inset></v-divider>
+        <v-card-title class="listItem">
+          <v-icon class="cyan--text text--darken-2 avatar">mdi-ip-network-outline</v-icon>
+          <v-flex class="pa-0">
+            <span class="grey--text">Available Load Balancer Classes</span><br>
+            <lb-class
+            v-for="lbClass in shootLoadbalancerClasses"
+            :lbClass="lbClass"
```
Comment (holgerkoser) [inline frontend/src/components/ShootDetails/ShootInfrastructureCard.vue]: split prop object into multiple props

File: frontend/src/components/ShootDetails/ShootInfrastructureCard.vue [hunk 4]
```diff
@@ -144,6 +163,16 @@ export default {
       }
       return `*.ingress.${this.shootDomain}`
     },
+    shootLoadbalancerClasses () {
+      const cloudProfile = this.cloudProfileByName('this.shootCloudProfileName')
```
Comment (holgerkoser) [inline frontend/src/components/ShootDetails/ShootInfrastructureCard.vue]: remove quotes

File: frontend/src/components/ShootDetails/ShootInfrastructureCard.vue [hunk 5]
```diff
@@ -144,6 +163,16 @@ export default {
...
+      const profileFloatingPools = get(cloudProfile, 'data.providerConfig.constraints.floatingPools')
+      const shootFloatingPoolName = get(this.shootItem, 'spec.provider.infrastructureConfig.floatingPoolName')
+      const shootFloatingPool = find(profileFloatingPools, { name: shootFloatingPoolName })
+      if (shootFloatingPool) {
+        return shootFloatingPool.loadBalancerClasses
+      }
+      return undefined
```
Comment (holgerkoser) [inline frontend/src/components/ShootDetails/ShootInfrastructureCard.vue]: try to remove

File: frontend/src/components/ShootDetails/LbClass.vue (hunk 2 above)
Comment (grolu) [reply frontend/src/components/ShootDetails/LbClass.vue]: content-class not available

**Pull request merged**